sdk/farmer/src/lib.rs

```diff
@@ -391,8 +410,37 @@ impl Config {
             replotting_thread_pool_size,
             sector_downloading_concurrency,
             sector_encoding_concurrency,
+            metrics_endpoints,
         } = self;
 
+        let mut prometheus_metrics_registry = Registry::default();
```

Comment:
Again, you shouldn't create multiple registries, it should be one for the whole app that is threaded as an argument throughout. See now `subspace-farmer` does it, this registry should probably be an argument just like in networking.